man/systemd.unit.xml

```diff
@@ -2098,6 +2098,16 @@ Note that this setting is <emphasis>not</emphasis> influenced by the <varname>Us
           <xi:include href="standard-specifiers.xml" xpointer="V"/>
           <xi:include href="standard-specifiers.xml" xpointer="w"/>
           <xi:include href="standard-specifiers.xml" xpointer="W"/>
+          <row>
+            <entry><literal>%y</literal></entry>
+            <entry>The path to the fragment</entry>
+            <entry>This is the path where the main part of the unit file is located. For linked unit files, the real path outside of the unit search directories is used. For units that don't have a fragment file, this specifier will return an error.</entry>
```

Comment:
What does "return an error" look like for a user with that in the unit? Will it fail to start?

Reply:
Yes. It fails like with a bad specifier, e.g. %X (which is not defined).
In the first version I made %y and %Y return an empty string, but that seems like a pitfall. I think it's better to fail, then e.g. try to write to `%y/output`, which would then become `/output`.

Reply:
Sounds good - can you maybe change this to say "will raise an error"? 'Return' sounded a bit confusing to me, since it's not something you call

Reply:
Done.